R/birdlife.R

```diff
@@ -25,3 +25,46 @@ birdlife_habitat = function(id){
   out = cbind(id, tables[[habitat_table_number]])
   out[-nrow(out), ] # Drop last row (altitude)
 }
+
+
```

Comment:
Ok.  I wasn't going to bother with roxygen until it was reviewed, but I'll do that now.